src/Events.Functions/Services/CertificateResolverService.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Functions.Configuration;
+using Altinn.Platform.Events.Functions.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Functions.Services
+{
+    /// <summary>
+    /// Class to resolve certificate to generate an access token
+    /// </summary>
+    /// <remarks>This class is excluded from code coverage because it has no logic to be tested.</remarks>
+    [ExcludeFromCodeCoverage]
```

Comment:
This class could be tested. The reason we don't have testing of the KeyVaultService is the heavy dependence on KeyVault clients. The `remarks` is misleading in that regard.